<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateManualDotColorMatchingDialog</class>
 <widget class="QDialog" name="CreateManualDotColorMatchingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_43">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>探
索</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>1</number>
    </property>
   </widget>
   <widget class="QFrame" name="frame_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label_44">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>20</width>
       <height>61</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>輝
度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QSpinBox" name="EditSelfSearch">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>ブロック単体での探索ドット数</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="ButtonOK">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Qtres/OK.png</normaloff>:/Qtres/OK.png</iconset>
   </property>
  </widget>
  <widget class="QFrame" name="frame_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>371</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;検&lt;/p&gt;&lt;p&gt;出&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>8</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>明側</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>28</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>OK輝度範囲 R</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>98</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>OKドット数</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>8</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>暗側</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>OK輝度範囲 G</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>OK輝度範囲 B</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QSpinBox" name="EditOKDotB">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>99999999</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="EditBrightWidthBGL">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="EditBrightWidthBBL">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="EditBrightWidthBBH">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="EditBrightWidthBRL">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-2</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="EditBrightWidthBGH">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="EditBrightWidthBRH">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>-2</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QSpinBox" name="EditOKDotN">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>72</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>99999999</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="EditBrightWidthNGL">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>22</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="EditBrightWidthNBL">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>42</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="EditBrightWidthNBH">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>42</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="EditBrightWidthNRL">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="EditBrightWidthNGH">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>22</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="EditBrightWidthNRH">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>0</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QToolButton" name="toolButtonBroad">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonNarrow">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>N</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameLibFolder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>460</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Qtres/Cancel.png</normaloff>:/Qtres/Cancel.png</iconset>
   </property>
  </widget>
  <widget class="QFrame" name="frame_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_41">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>補
正</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>1</number>
    </property>
   </widget>
   <widget class="QFrame" name="frame_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label_42">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>20</width>
       <height>61</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>輝
度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QSpinBox" name="EditAdjustBlack">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditAdjustWhite">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>明側　最大補正輝度量</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>暗側　最大補正輝度量</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="EditLibID">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetLibList">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>201</width>
     <height>141</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="EditLibName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
